crates/swc_css_parser/src/lib.rs

```diff
@@ -37,28 +37,6 @@ where
     }
 }
 
-/// Parse a given string as `T`.
-///
-/// If there are syntax errors but if it was recoverable, it will be appended
-/// to `errors`.
-pub fn parse_str<'a, T>(
-    src: &'a str,
-    start_pos: BytePos,
-    end_pos: BytePos,
-    config: ParserConfig,
-    errors: &mut Vec<Error>,
-) -> PResult<T>
-where
-    Parser<Lexer<StringInput<'a>>>: Parse<T>,
-{
-    let lexer = Lexer::new(StringInput::new(src, start_pos, end_pos), config);
```

Comment:
I think we don't need it:
- `ecma` and `html` doesn't support it in favor `FileName::Anon` for `SourceFile`